### PR TITLE
a31-weather

### PR DESCRIPTION
**Pull request by valentinavargas31**

### CODE REVIEW

File: src/mm/html-css/mv-A31-Weather/page1.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+pagina1
```
[the remark this answers is not in the document]
Comment (valentinavargas31) [reply src/mm/html-css/mv-A31-Weather/page1.html]: corrected

File: src/mm/html-css/mv-A31-Weather/page2.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    <link rel="stylesheet" href="page2.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+  </head>
+  <body>
+    <div class="blue">
+      <div class="head">
+        <a
```
[the remark this answers is not in the document]
Comment (valentinavargas31) [reply src/mm/html-css/mv-A31-Weather/page2.html]: corrected

File: src/mm/html-css/mv-A31-Weather/page2.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
+body {
```
[the remark this answers is not in the document]
Comment (valentinavargas31) [reply src/mm/html-css/mv-A31-Weather/page2.css]: corrected

File: src/mm/html-css/mv-A31-Weather/page1.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    />
+  </head>
+  <body>
+    <div class="content">
+      <div class="header">
+        <button id="button1">::</button>
+        <div><i class="fa-solid fa-location-dot"></i> <span>Minsk</span></div>
+        <i class="fa-solid fa-ellipsis-vertical"></i>
+      </div>
+      <button>Updating</button> <br />
```
[the remark this answers is not in the document]
Comment (valentinavargas31) [reply src/mm/html-css/mv-A31-Weather/page1.html]: corrected

File: src/mm/html-css/mv-A31-Weather/page2.css [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  justify-content: center;
+  gap: 20px;
+  line-height: 0;
+}
+
+img {
+  width: 200px;
+}
+
+#p1 {
```
[the remark this answers is not in the document]
Comment (valentinavargas31) [reply src/mm/html-css/mv-A31-Weather/page2.css]: I already corrected them and made them more descriptive

**Review by Iregui44 — APPROVED** [review]

**Pull request merged**